Tema5/positioning.css

```
@@ -0,0 +1,41 @@
+div {
+    border-radius: 10px;
+    border-style: dotted;
+    border-color: rgb(131, 95, 116);
+    text-align: center;
+    text-transform: uppercase;
+}
+.parent {
+    background-color:rgb(187, 201, 184);
+    border-width: 5px;
+    height: 280px;
+    width: 750px;
+}
```

Ar trebuii adaugat un position pentru parinte pentru ca div-urile copil sa fie positionate relativ la parinte;

```suggestion
.parent {
    background-color:rgb(187, 201, 184);
    border-width: 5px;
    height: 280px;
    width: 750px;
    position: relative;
}
```